### PR TITLE
update DateRangeSelector

### PR DESCRIPTION
**Pull request by nuria-balaguer**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ export {
 export {
     default as TemperatureChart2,
 } from './TemperatureChart2'
+export {
+    default as DateRangeSelector,
+} from './DateRangeSelector'
```
Comment (dimaPortyanka) [inline src/index.js:9]: please update dts file

File: src/index.js [hunk 2]
```diff
@@ -40,6 +43,3 @@ export {
 export {
     default as ErrorBoundary,
 } from './ErrorBoundary'
-export {
-    default as I18nLocaleProvider,
-} from './I18nLocaleProvider'
```
Comment (dimaPortyanka) [inline src/index.js:45]: nope :) this is bad :)

File: src/DateSelect/DateSelect.js [hunk 3]
```diff
@@ -58,12 +57,12 @@ const DateSelect = (props) => {
     } = props
 
     const handleOnChange = useCallback((date) => {
-        const newValue = dateToDateStr(date)
+        const newValue = new Date(date)
```
Comment (dimaPortyanka) [inline src/DateSelect/DateSelect.js:60]: would be good to handle invalid date case

File: src/DateSelect/__tests__/DateSelect.spec.js [hunk 4]
```diff
@@ -2,19 +2,86 @@ import React from 'react'
...
+} from '@material-ui/pickers'
 
 import DateSelect from '../DateSelect'
 
 describe('shared-components/DateSelect', () => {
     it('should match snapshot', () => {
         const wrapper = shallow(
             <DateSelect
-                value="26.06.2020"
+                value="10.6.2018"
```
Comment (ricardovoegeli) [inline src/DateSelect/__tests__/DateSelect.spec.js:15]: I think should be 10.06.2018

**Review by dimaPortyanka — DISMISSED** [review]

**Review by AnastasiiaManilnikova — DISMISSED** [review]

**Review by AnastasiiaManilnikova — APPROVED** [review]

**Review by dimaPortyanka — APPROVED** [review]

File: src/DateSelect/__tests__/DateSelect.spec.js (hunk 4 above)
Comment (nuria-balaguer) [reply src/DateSelect/__tests__/DateSelect.spec.js]: done

**Pull request merged**